index.html

```diff
@@ -0,0 +1,448 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <div class="menu-bars">
+        <i class="fas fa-bars open-btn"></i>
+        <i class="fas fa-times close-btn"></i>
+      </div>
+
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href=".projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href=".contact"
```

Comment:
When you click the 'Portfolio' and 'Contact' on the menu options, the correct part of the page is not displayed. This can happen when you input a wrong or invalid href attribut. Kindly check and change the attribute value of href.

Reply:
Thanks for the professional review @jerryowusu 
I'm going through it now